src/client/getOrCreateEditor.js

```diff
@@ -31,7 +31,7 @@ export const getOrCreateEditor = ({
   // The path for this file in the ShareDB document.
   const path = [fileId, 'text'];
 
-  const extensions = [
+  let extensions = [
```

Comment:
I don't see the need for `let` here.
```suggestion
  const extensions = [
```